x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/method_connector/add_connector_package_logic.test.ts

```diff
@@ -57,7 +57,7 @@ describe('AddConnectorPackageLogic', () => {
         AddConnectorPackageApiLogic.actions.apiSuccess({ indexName: 'success' } as any);
         await nextTick();
         expect(flashSuccessToast).toHaveBeenCalledWith('Index created successfully', {
-          text: 'You can use a connector build a search experience for your new Elasticsearch index.',
+          text: 'You can use a connector to build a search experience for your new Elasticsearch index.',
```

Comment:
Checking text content is almost as bad as checking the number of elements!!!!

Reply:
Fair!